meta-mender-demo/recipes-extended/images/mender-demo-full-delta.bb

```diff
@@ -0,0 +1,9 @@
+require recipes-extended/images/mender-demo-full-rw.bb
+
```

Comment:
I think we should be more future ready and name this image `mender-demo-full-rw-commercial`, having delta updates and later on mender-monitor and any other commercial component that we add.